web_easy_switch_company/__openerp__.py

```diff
@@ -22,7 +22,7 @@
 
 {
     'name': 'Multicompany - Easy Switch Company',
-    'version': '8.0.1.0.0',
+    'version': '9.0.1.0.0',
     'category': 'web',
     'description': """
 Add menu to allow user to switch to another company more easily
```

Comment:
Move this description to a README.rst file and use the OCA's latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst